src/main/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectApiRestClient.java

```diff
@@ -119,8 +122,27 @@ public TagApi tag(String ref) throws RestApiException {
         JsonElement tags = gerritRestClient.getRequest(request);
         return tagInfoParser.parseTagInfos(tags);
     }
-
-
+    
+    /* get accesses of the project
```

Comment:
You can remove this comment - it's also not there for other methods in this class. The interface should document it.